Lib/ufo2ft/postProcessor.py

```diff
@@ -98,6 +99,13 @@ def process(
           is True and CFF or CFF2 table is present. Choose between "cffsubr" or
           "compreffor". By default "cffsubr" is used for both CFF 1 and CFF 2.
           NOTE: compreffor currently doesn't support input fonts with CFF2 table.
+
+        compileTrueTypeHinting:
+          Compile TrueType hinting from the UFO, if present. You must make sure
+          that the outlines from the UFO have not been altered before this
+          step. The point indices change e.g. when removing overlaps or
+          changing the path directions. This will most certainly make any
+          hinting data in the source UFO invalid.
```

Comment:
if that's the case, then we should probably warn somewhere if both compileTrueTypeHinting and other incompatible options are set